shipyard-controller/handler/shipyardcontroller.go

```diff
@@ -150,33 +151,40 @@ func (sc *shipyardController) HandleIncomingEvent(event models.Event, waitForCom
 			if err != nil {
 				log.WithError(err).Error("Unable to handle sequence '.triggered' event")
 			}
-			complete(err)
-		}()
-	case string(common.StartedEvent):
-		go func() {
-			err := sc.handleTaskStarted(event)
-			if err != nil {
-				log.WithError(err).Error("Unable to handle task '.started' event")
-			}
-			complete(err)
+			cb(err)
 		}()
-	case string(common.FinishedEvent):
+	case string(common.StartedEvent), string(common.FinishedEvent):
```

Comment:
at this level no distinction between started/finished events is required anymore